measurements/openapi/measurements.yml

```diff
@@ -249,6 +249,161 @@ paths:
         default:
           description: Default response
 
+  /aggregation:
+    get:
+      summary: ''
+      description: ''
+      operationId: 'measurements.api.get_aggregated'
+      parameters:
+        - name: input
+          in: query
+          type: string
+          minLength: 3
+          description: The input (for example a URL or IP address) to search measurements for
+
+        - name: domain
+          in: query
+          type: string
+          minLength: 3
+          description: The domain to search measurements for
+
+        - name: probe_cc
+          in: query
+          type: string
+          description: The two letter country code
+
+        - name: probe_asn
+          in: query
+          type: string
+          description: the Autonomous system number in the format "ASXXX"
+
+        - name: test_name
+          in: query
+          type: string
+          description: The name of the test
+          enum:
+          - web_connectivity
+          - http_requests
+          - dns_consistency
+          - http_invalid_request_line
+          - bridge_reachability
+          - tcp_connect
+          - http_header_field_manipulation
+          - http_host
+          - multi_protocol_traceroute
+          - meek_fronted_requests_test
+          - whatsapp
+          - vanilla_tor
+          - facebook_messenger
+          - ndt
+          - dash
+          - telegram
+          - psiphon
+          - tor
```

Comment:
Maybe we want to define this as a special type so that we don't have to update the test_name list everywhere each time we add a new one.